examples/servicediscovery/privatednsnamespace.yaml

```
@@ -3,11 +3,14 @@ kind: PrivateDNSNamespace
 metadata:
   name: example-privatednsnamespace
 spec:
+  providerConfigRef:
+    name: example
   forProvider:
-    region: eu-central-1
+    region: us-east-1
     description: "test"
     name: "test"
-    vpc: default
+    # The ID of the Amazon VPC that you want to associate the namespace with
+    vpc: vpc-02504719d1cb7a9aa
```

```suggestion
    vpcIdRef:
      name: sample-vpc
```

Can we reference to VPC example YAML here so that we're closer to getting `kubectl apply -f examples` to work?

---

This was also something that I considered, however, it looks like this is not supported by `PrivateDNSNamespace` API: https://github.com/crossplane/provider-aws/blob/72d273177575641f74fe5d225848f2d3dd09e2fe/apis/servicediscovery/v1alpha1/zz_private_dns_namespace.go#L49

---

Interesting. If we weren't thinking of making references more generic (per https://github.com/crossplane/crossplane/issues/1770) it would probably make sense to require the existence of such references part of the conformance test suite.